msa/black-box-tests/src/test/java/org/thingsboard/server/msa/ui/base/AbstractDriverBaseTest.java

```diff
@@ -67,7 +66,7 @@ abstract public class AbstractDriverBaseTest extends AbstractContainerTest {
 
     @SneakyThrows
     @BeforeMethod
-    public void openBrowser() {
+    public void startUp() {
         log.info("===>>> Setup driver");
         testRestClient.login(TENANT_EMAIL, TENANT_PASSWORD);
         ChromeOptions options = new ChromeOptions();
```

Comment:
Why do we need open/close the browser for each test? Its expensive operation and users likely do not do the browser restart on every action.